helpers/query/order.js

```diff
@@ -6,7 +6,19 @@ helpers.register('order', function(order, values, query){
 
   if (typeof order === 'string') return output + order;
 
-  if (Array.isArray(order)) return output + order.join(', ');
+  if (Array.isArray(order)) {
+  	if (typeof order[0] === 'string') {
+  		return output + order.join(', ');	
+  	} else {
+		for (var i in order) {
+		  for (var key in order[i]){
+			output += utils.quoteObject(key, query.__defaultTable) + ' ' + order[i][key] + ', ';
+		  }
+		}
+		return output.substring(0, output.length - 2);
+  	}
+  	
+  }
```

Comment:
This looks good to me, we'll just need to make the tabs consistent (I know, I know nitpick). Just use two spaces and in place of tabs and I'll merge and get this pushed to npm